tests/test_chart_sync.py

```diff
@@ -381,6 +405,20 @@ def test_chart_updated_manifest_inside_path(empty_git_repo, requests_mock):
     assert re.search(CHART_RELEASE_COMMIT_RE, _last_commit())
 
 
+def test_does_not_crash_for_multidoc(empty_git_repo, requests_mock):
+    mkdir("deploy")
+    _add_manifest(MANIFEST_WITH_MULTIPLE_DOCUMENTS, path="deploy/1-multi.yaml")
+    _add_manifest(MANIFEST_WITH_SINGLE_IMAGE, path="deploy/2-helmrelease.yaml")
+    _init_commit()
+    requests_mock.get(HELM_REPO_INDEX, text=CHART_REPO_INDEX_WITH_NEW_CHARTS)
+
+    updater = Updater(Git(empty_git_repo), HelmRepo(HELM_REPO_URL))
+    updater.update_loop(one_shot=True)
+
+    assert _get_manifest("deploy/2-helmrelease.yaml") == UPDATED_MANIFEST_WITH_SINGLE_IMAGE
+    assert re.search(SINGLE_IMAGE_RELEASE_COMMIT_RE, _last_commit())
```

Comment:
consider also checking that the `MANIFEST_WITH_MULTIPLE_DOCUMENTS` was not changed

Reply:
OK, added.